user-identity-be/src/test/java/pl/larp/larpex/usi/user/adapter/memory/InMemoryUsersRepository.java

```diff
@@ -4,11 +4,13 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
+import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 import pl.larp.larpex.usi.user.domain.model.User;
 import pl.larp.larpex.usi.user.domain.port.IUsers;
 
 @Component
+@Profile("test") // TODO move to tests
```

Comment:
todo

Reply:
yup and it will stay that way until DB is going to be split up

Reply:
I don't have the strength in me to add an issue number